modules/ipi-install-removing-the-provisioner-node.adoc

```diff
@@ -0,0 +1,17 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-install-removing-the-provisioner-node_{context}"]
+= Removing the provisioner node
+
+Optionally, the provisioner node can be removed after installation.
+
+.Procedure
+
+Before removing the provisioner node, verify that you have a backup of the following:
+
+. The `install-config.yaml` file in case the cluster must be redeployed.
```

Comment:
Doesn't install-config.yaml get deleted by the installer? I think it's too late to grab a backup by this point.